sql_generators/active_users_deletion_requests/templates/mobile_deletion_request_query.sql

```diff
@@ -0,0 +1,237 @@
+WITH baseline AS (
+  SELECT
+    submission_date,
+    joined.normalized_channel,
+    client_id,
+    days_since_seen,
+    days_seen_bits,
+    days_created_profile_bits,
+    durations,
+    joined.normalized_os,
+    joined.normalized_os_version,
+    locale,
+    city,
+    country,
+    app_display_version,
+    device_model,
+    first_seen_date,
+    submission_date = first_seen_date AS is_new_profile,
+    uri_count,
+    is_default_browser,
+    CAST(NULL AS string) AS distribution_id,
+    isp,
+    IF(
+      isp = 'BrowserStack',
+      CONCAT('{{ app_value }}', ' BrowserStack'),
+      '{{ app_value }}'
+    ) AS app_name
+  FROM
+    `{{ project_id }}.{{ app_name }}.clients_last_seen_joined` AS joined
+  INNER JOIN
+    `{{ app_name }}.deletion_request` AS request
+  ON
+    client_info.client_id = client_id
+  WHERE
+    joined.submission_date <= @submission_date
+    AND DATE(request.submission_timestamp)
+    BETWEEN DATE_SUB(@submission_date, INTERVAL 28 DAY)
```

Comment:
@relud the parameters were not read using the macro so I'd propose ton use the filter in the query to run 28 days before and run in the DAG every 28 days. Does that work or would be disaligned with shredder?

Reply:
i don't understand what is meant by `the parameters were not read using the macro`

Reply:
> using {% raw %} is for the YAML to parse correctly.

you definitely don't need `{% raw %}` for the yaml to parse correctly, e.g. https://github.com/mozilla/bigquery-etl/blob/360b34252f4e1a36960e713b4497e62e9c47f04f/sql/moz-fx-data-shared-prod/monitoring_derived/stable_and_derived_table_sizes_v1/metadata.yaml#L13 and https://github.com/mozilla/bigquery-etl/blob/360b34252f4e1a36960e713b4497e62e9c47f04f/sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_etl_sql_run_check_v1/metadata.yaml#L13-L15

Reply:
Strangely, we debug this error with Kik and it was failing on parsing the YAML.. Do we definitely need to set the 28 days in the scheduling, not in the query?

Reply:
The problem happens during the sql generation, I moved it to the template.